zh/deploy-tidb-binlog.md

```diff
@@ -278,3 +278,71 @@ tlsSyncer: {}
     #  - TiDB
 ...
 ```
+
+## 缩容/移除 Pump/Drainer 节点
+
+如需详细了解如何维护 binlog 集群节点状态信息，可以参考 [这里](https://docs.pingcap.com/zh/tidb/stable/maintain-tidb-binlog-cluster#pumpdrainer-的启动退出流程)。
```

Comment:
```suggestion
如需详细了解如何维护 TiDB Binlog 集群节点状态信息，可以参考 [Pump/Drainer 的启动、退出流程](https://docs.pingcap.com/zh/tidb/stable/maintain-tidb-binlog-cluster#pumpdrainer-的启动退出流程)。
```
It is recommended to give readers some hint about what they will see in the linked page.